lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -7,6 +7,7 @@ def self.included(base)
       
       # Add ability to skip callbacks for save/update.
       def self.skip_callbacks
+        _execute_position_update!
```

Comment:
Is this intentional here? I don't see the `sql` parameter being passed in.

Reply:
Ooo, no. Good call. Must've done a weird copy paste thing. I'm just going to rebase that out.